optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/service/region/BoundingBoxTest.java

```diff
@@ -65,4 +67,19 @@ void should_fail_if_bounding_box_has_zero_dimension() {
                 Coordinates.valueOf(1.0, 10.0)
         )).withMessageMatching(".*\\(10\\.0E.*\\(10\\.0E.*");
     }
+
+    @Test
+    void constructor_args_not_null() {
+        assertThatNullPointerException().isThrownBy(() -> new BoundingBox(null, Coordinates.valueOf(1.0, 1.0)));
+        assertThatNullPointerException().isThrownBy(() -> new BoundingBox(Coordinates.valueOf(1.0, 1.0), null));
+    }
+
+    @Test
+    void should_work_with_southwest_and_northeast() {
```

Comment:
What happens if you have a north-west coordinate and a south-east coordinate (ex: (10, -10), (-10, 10) )

Reply:
That's not allowed and is covered by a negative test that has existed before this PR. I've just added a positive test to kill more mutations.